components/Chips/tests/snapshot/MDCChipFieldSnapshotTests.m

```diff
@@ -108,6 +108,21 @@ - (void)testTextFieldWithZeroMinimumWidthIsOnSameLineAsWideChips {
   [self snapshotVerifyView:self.chipField];
 }
 
+- (void)testEmptyPlaceholderSizeDoesNotAccountForContentInsets {
```

Comment:
Will this test title be accurate once the issue is fixed? I'd expect this test to eventually be "testEmptyPlaceholderSizeAccountsForContentInsets"; may be better to have the right name with the wrong screenshot so that when you fix it the screenshot matches the name.

Reply:
Yep! Right now, this title is inaccurate and is exposing a bug. An empty placeholder should not account for insets in its requested size, but should only require the minimum hit target size. A PR to fix this will be published immediately after this one lands :)